tests/testthat/testImportSNV.TCGA.R

```diff
@@ -0,0 +1,73 @@
+# testImportSNV.TCGA.R
+
+context("Tools for importing MAF files into a rSNV file")
+
+test_that("parameter errors are correctly handled", {
+    # fMAF is not provided
+    # fMAF is NULL
+    # fMAF input vector is empty
+    # fMAF input vector contains non-string values
+    # file in fMAF could not be opened/does not exist
+    # if rSNV parameter is provided, rSNV file does not exist
+    # if rSNV parameter is provided, but is NULL
+    # if rSNV parameter is provided, but is non-string value
+    # if rSNV is not provided, new file could not be created (file name conflict)
+    # if na.rm is not given logical input
+    # if silent is not given logical input
+    # write.log is not given logical input
+})
+
+test_that("check various forms of input parameters work correctly", {
```

Comment:
I don't see that you can check that without running the actual code. I think these tests need to be merged with the next block ("works correctly")